_source/_posts/2023-03-22-stepup-okta.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: blog_post
+title: "Step-Up Authentication Examples With Okta"
+author: indranil-jha
+by: advocate
+communities: [security,mobile]
+description: "Understand Step Up Authentication in Okta's Workforce Identity Cloud"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/step-up-okta/stepup.jpg
+type: awareness
+---
+
+# Step-up Authentication with Okta Workforce Identity Cloud
+
+Step-up authentication in an application is a pattern of allowing access to non-critical resources using a basic level of authentication, and requiring additional authentication for critical resources. This offers a balance between frictionless user experience and security in an application.
+
+There are several methodologies to provide the functionality as discussed [here](/blog/2023/03/08/step-up-auth). 
+
+In this article we will explore few ways [Okta Workforce Identity Cloud (WIC)](https://www.okta.com/workforce-identity/) can support step-up authentication in an application.
+
+
+## Modern Application using OpenID Connect (OIDC)
```

Comment:
The h2 and h3 tags are used for SEO so each section should enforce the keywords and terms we want to apply towards SEO